report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -37,13 +37,13 @@ \section*{Rust}
 Rust is a multi-paradigm, general-purpose programming language originally developed by Graydon Hoare in 2006 as a personal project at Mozilla.
 The project was then sponsored by Mozilla from 2009 to 2021 where the project along with all trademarks was moved to the newly create Rust Foundation.
 The idea behind Rust was to challenge the idea that high-level ergonomics and low-level control come at the cost of each other\cite{Rust_Book}.
-Most other programming languages today operate in a band where either safety or control is valued highest and the other diminished. An example of this is C that has high level control but lacks safety.\todo{source}
+Most other programming languages today operate in a band where either safety or control is valued highest and the other diminished. An example of this is C that has high level control but lacks safety.
 Here Rust provides the low-level power of C along with the solid safety features programmers have gotten used to in high-level languages like Python.
-As such, the main overall goal of Rust is to provide as much safety as possible in a low-level programming language without sacrificing any speed\cite{Rust_in_Action}.
+As such, the main overall goal of Rust is to provide as much safety as possible in a low-level programming language without sacrificing any speed. \cite{Rust_in_Action}
```

Comment:
Is this a good enough source? Does Rust in Action say that "C has high level control but lacks safety."?